Foundation/NSObjCRuntime.swift

```diff
@@ -196,6 +196,9 @@ internal func NSRequiresConcreteImplementation(_ fn: String = #function, file: S
 }
 
 internal func NSUnimplemented(_ fn: String = #function, file: StaticString = #file, line: UInt = #line) -> Never {
+    #if os(Android)
+    NSLog("\(fn) is not yet implemented. \(file):\(line)")
+    #endif
```

Comment:
It should but this is a Foundation PR. I’ll take a look. Annoyingly using __android_log_print requires a -llog when you link so it wouldn’t be a trivial change.